src/Kafka/Metrics/components/PartitionCard.tsx

```diff
@@ -0,0 +1,125 @@
+import {
+  Bullseye,
+  Card,
+  CardBody,
+  CardTitle,
+  Skeleton,
+  TextContent,
+  Title,
+  Text,
+  CardFooter,
+  Alert,
+  Stack,
+  StackItem,
+  AlertVariant,
+} from "@patternfly/react-core";
+import {
+  ExclamationCircleIcon,
+  ExclamationTriangleIcon,
+} from "@patternfly/react-icons";
+import { VoidFunctionComponent } from "react";
+import { useTranslation } from "react-i18next";
+import { ChartPopover } from "./ChartPopover";
+import { EmptyStateNoMetricsData } from "./EmptyStateNoMetricsData";
+
+type PartitionCardProps = {
+  metric: string | number | undefined;
+  isLoading: boolean;
+};
+
+export const PartitionCard: VoidFunctionComponent<PartitionCardProps> = ({
+  metric,
+  isLoading,
+}) => {
+  const { t } = useTranslation();
+
+  return (
+    <Card
+      isFullHeight
+      data-testid={t("metrics:metric_kpi_topicPartitions_name")}
+    >
+      <CardTitle component="h3">
+        {t("metrics:metric_kpi_topicPartitions_name")}{" "}
+        <ChartPopover
+          title={t("metrics:metric_kpi_topicPartitions_name")}
+          description={t("metrics:metric_kpi_topicPartitions_description")}
+        />
+      </CardTitle>
+      <CardBody>
+        <Bullseye>
+          {!isLoading &&
+            (metric === undefined ? (
+              <EmptyStateNoMetricsData />
+            ) : (
+              <Stack hasGutter>
+                <StackItem>
+                  <Title
+                    headingLevel="h3"
+                    size="4xl"
+                    aria-valuetext={`${metric} ${t(
+                      "metrics:metric_kpi_topicPartitions_name"
+                    )}`}
+                  >
+                    {metric}{" "}
+                    {(() => {
+                      if (metric === 1000) {
+                        return (
+                          <ExclamationCircleIcon color="var(--pf-global--danger-color--100)" />
+                        );
+                      } else if (metric >= 950) {
```

Comment:
I think this line is determining when the warning message should display? 

Looking at the [design document](https://docs.google.com/presentation/d/11Hhh7elHwLE_rAYRVp8pm95leV6V3O9mcGSoKEdvBME/edit#slide=id.gf3db16a3cf_0_5), there's a note that the warning should display when the current value is 95% of the limit. 

Related to @riccardo-forina's previous feedback, these values `950` and `1000` shouldn't be hard-coded but should be coming from the API. Where you have `950`, we would actually want to determine what 95% of the limit provided by the API is.